src/layouts/Login/LoginPageSgid.tsx

```diff
@@ -0,0 +1,257 @@
+import {
+  Box,
+  Divider,
+  Flex,
+  GridItem,
+  GridProps,
+  Text,
+  Grid,
+  HStack,
+  VStack,
+  Tabs,
+  TabList,
+  TabPanels,
+  TabPanel,
+} from "@chakra-ui/react"
+import {
+  Button,
+  GovtMasthead,
+  Tab,
+  InlineMessage,
+  Link,
+} from "@opengovsg/design-system-react"
+import { useEffect, useState, PropsWithChildren, useMemo } from "react"
+import { useHistory, useLocation } from "react-router-dom"
+
+import { PRIVACY_POLICY_LINK, TERMS_OF_USE_LINK } from "constants/config"
+
+import { useGetSgidAuth, useLogin, useVerifyOtp } from "hooks/loginHooks"
+
+import { getAxiosErrorMessage } from "utils/axios"
+import { useErrorToast, useSuccessToast } from "utils/toasts"
+
+import { IsomerLogo, LoginImage, OGPLogo } from "assets"
+import { DEFAULT_RETRY_MSG } from "utils"
+
+import { LoginForm, LoginProps, OtpForm, OtpProps } from "./components"
+
+const LOGIN_GRID_LAYOUT: Pick<
+  GridProps,
+  | "gridTemplateAreas"
+  | "gridTemplateColumns"
+  | "gridTemplateRows"
+  | "height"
+  | "width"
+> = {
+  gridTemplateAreas: `"image content" 
+                      "credits links"`,
+  gridTemplateColumns: "2fr 4fr",
+  gridTemplateRows: "1fr 5rem",
+  height: "100%",
+  width: "100%",
+}
+
+interface FooterLinkProps {
+  link: string
+}
+
+const FooterLink = ({
+  link,
+  children,
+}: PropsWithChildren<FooterLinkProps>): JSX.Element => (
+  <Link
+    href={link}
+    isExternal
+    textDecorationLine="none"
+    color="text.link.dark"
+    _hover={{
+      color: "text.link.dark",
+      textDecorationLine: "underline",
+    }}
+  >
+    {children}
+  </Link>
+)
+
+const LoginContent = (): JSX.Element => {
```

Comment:
it seems like the crux of sgid is just the api calls, which we should just abstract out as a hook? 

the component (`LoginPageSgid`) should be removed isn't it?

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1369/commits/b922ca67e5bd7957924ad4a60ee74dff4dbadf1a